frame/support/test/tests/reserved_keyword/on_initialize.stderr

```diff
@@ -1,47 +1,39 @@
 error: Invalid call fn name: `on_finalize`, name is reserved and doesn't match expected signature, please refer to `decl_module!` documentation to see the appropriate usage, or rename it to an unreserved keyword.
   --> $DIR/on_initialize.rs:30:1
    |
-30 | reserved!(on_finalize on_initialize on_finalise on_initialise offchain_worker deposit_event);
```

Comment:
Is this removal of reserved functions intentional?

Reply:
yes he wrote:

> I have also removed traces of on_finalise and on_initialise as they have been deprecated since v1.0.